packages/components/src/border-box-control/stories/index.js

```diff
@@ -85,6 +85,7 @@ Default.args = {
 		width: '1px',
 	},
 	__next36pxDefaultSize: false,
+	popoverPlacement: 'right-start',
```

Comment:
I added `popoverPlacement` while testing for the [stale `anchor` value](https://github.com/WordPress/gutenberg/pull/43789/files#r960960748), and decided to keep it for convenience.

But the component's stories definitely need to be rewritten in TypeScript so that they can show all props

Reply:
I have it on my radar to rewrite the border control component stories in TypeScript and update them to follow the latest approach and guidelines. Unfortunately, I haven't had the bandwidth in the lead-up to 6.1 but hope to in the not-too-distant future.